### ISSUE TITLE
Members's page refactoring

### ISSUE THREAD

**Issue by SimonNtz** [SimonNtz/DAL_web#18]
- [ ] Reduce photos's size

- [ ] Restructure list-group to have two  list-group per row.

**Comment by rspt** [comment]
> Restructure list-group to have two list-group per row.

Even in mobile size?

**Comment by SimonNtz** [comment]
Not needed in mobile size.

**Comment by rspt** [comment]
What is not needed in mobile size??? We need to support all screen's sizes ;)

My question was if we display members two by two in small width screen.

**Comment by SimonNtz** [comment]
Yes that what I meant ! One by one with small mobile screen. I will create
a new issue to resize the image they are too big.

2017-10-22 13:11 GMT+02:00 Julien Grünhagel <notifications@github.com>:
